crates/vm/src/arch/config.rs

```diff
@@ -54,11 +54,13 @@ pub struct MemoryConfig {
     pub decomp: usize,
     /// Maximum N AccessAdapter AIR to support.
     pub max_access_adapter_n: usize,
+    /// An expected upper bound on the number of memory accesses.
+    pub access_capacity: usize,
 }
 
 impl Default for MemoryConfig {
     fn default() -> Self {
-        Self::new(29, 1, 29, 29, 17, 64)
+        Self::new(29, 1, 29, 29, 17, 64, 1 << 24)
```

Comment:
2**24 seems pretty high, maybe just 2**20?

Reply:
This is for memory. If a chip has up to 2^20 records and we have dozens of chips and each record does several accesses, then 2^24 might still be small?

Reply:
can you lmk the execution performance diff (via criterion bench) between 2^20 and 2^24, just for a sense?

Reply:
from 1 << 20 to 1 << 24 (regex_execute bench):
```
regex/execute           time:   [377.90 ms 391.66 ms 405.92 ms]
                        change: [-11.309% -6.3323% -0.8434%] (p = 0.05 < 0.05)
                        Change within noise threshold.
```
on the border of statistical signficance